packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -5723,6 +5763,184 @@ class _ScribbleFocusableState extends State<_ScribbleFocusable> implements Scrib
   }
 }
 
+class _Scribe extends StatefulWidget {
+  const _Scribe({
+    required this.child,
+    required this.editableKey,
+    required this.enabled,
+    required this.focusNode,
+    required this.selectionControls,
+  });
+
+  final Widget child;
+  final GlobalKey editableKey;
+  final bool enabled;
+  final FocusNode focusNode;
+  final TextSelectionControls? selectionControls;
+
+  @override
+  State<_Scribe> createState() => _ScribeState();
+}
+
+class _ScribeState extends State<_Scribe> {
+  // The handwriting bounds padding of EditText in Android API 34.
+  static const EdgeInsets _handwritingPadding = EdgeInsets.symmetric(
```

Comment:
Is this a fixed size or just the size of the input decorator?

Reply:
This is the fixed amount of padding that native Android uses to detect Scribe events outside of the field.